scripts/data/build_hardcoded.py

```diff
@@ -0,0 +1,354 @@
+import argparse
+import os
+import logging
+from functools import partial
+from datasets import load_dataset, Dataset, DatasetDict
+from huggingface_hub import HfApi
+
+"""
+Example use, for Tülu 3, run the following:
+
+python scripts/data/build_hardcoded.py \
+    --filter_tags olmo \
+    --model_name "Tülu 3" \
+    --base_model "Llama 3.1" \
+    --posttrain_recipe "Tülu 3" \
+    --context_length 4096 \
+    --license "Llama 3.1 Community License Agreement" \
+    --weights_link "https://huggingface.co/allenai"
```

Comment:
Let's add `--target_namespace allenai` here by default it's a bit easier to copy and paste and edit.